azure-stack/hci/manage/arc-extension-management.md

```diff
@@ -73,6 +122,19 @@ To enable an automatic upgrade, navigate to the **Extensions** page and perform
 
     :::image type="content" source="media/arc-extension-management/arc-extension-enable-auto-upgrade-2.png" alt-text="Screenshot of the notification to enable auto upgrade in the Azure portal." lightbox="media/arc-extension-management/arc-extension-enable-auto-upgrade-2.png":::
 
+# [Azure CLI](#tab/azurecli)
+To install and enable auto upgrade for a specific extension like `AdminCenter` run the following command:
+    ```azurecli
+    extensionName="AzureMonitorWindowsAgent" # Replace with the extension  name
+    
```

Comment:
This code block is not showing up correctly, remove the empty space before az_ stack-hci extension list \

Reply:
Removed.